### PR TITLE
new API for list and cancel database operations

### PR DESCRIPTION
**Pull request by jiawang003**
new API for list and cancel database operations

### CODE REVIEW

**Comment by fearthecowboy** [comment]
ATTENTION REQUIRED.
-------------

We have changed the `default` branch in the `azure-rest-api-specs` repository to use the branch labeled `current`.

This PR is opened against the `master` branch.

You have two options: 
1. Keep this PR open, and complete the review process as normal, and once this PR is merged against the `master` branch, we (@olydis & @fearthecowboy)  will create a PR against the current branch and make any updates required to work in the newly reorganized default branch.  

2. Close this PR and create a new one. That would mean pulling in the changes from the new default branch `current` moving any files and switching to the configuration-based layout.  (You are more than welcome to choose option 2, but I don't expect anyone to do that)

Either way, the work you are currently doing against the previous default branch `master` will be the last PR against that branch that we will accept and all new PRs will go against the new default branch  `current`.

If you have any questions or concerns, ping @fearthecowboy or @olydis (or go ask on the `#swagger` channel in slack) ASAP and we can help you thru the process.

**Review by nathannfan — APPROVED** [review]
Leave a comment

**Review by jaredmoo — APPROVED** [review]

**Review by fearthecowboy — DISMISSED** [review]

**Comment by salameer** [comment]
Reviewer will be assigned very shortly

File: arm-sql/2017-03-01-preview/swagger/cancelOperations.json [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "operationId",
```
Comment (dsgouda) [inline arm-sql/2017-03-01-preview/swagger/cancelOperations.json]: nit: Please add a description for this parameter.

File: arm-sql/2017-03-01-preview/swagger/cancelOperations.json [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+            "format": "uuid"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
```
Comment (dsgouda) [inline arm-sql/2017-03-01-preview/swagger/cancelOperations.json]: nit: Please verify whether this is a long running operation

**Review by dsgouda — COMMENTED** [review]
Couple of minor comments aside, looks good.

File: arm-sql/2017-03-01-preview/swagger/cancelOperations.json (hunk 2 above)
Comment (jaredmoo) [inline arm-sql/2017-03-01-preview/swagger/cancelOperations.json]: It is not.

File: arm-sql/compositeSql.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "./2015-05-01-preview/swagger/virtualNetworkRules.json",
     "./2015-05-01-preview/swagger/servers.json",
     "./2015-05-01-preview/swagger/serverKeys.json",
-    "./2015-05-01-preview/swagger/encryptionProtectors.json"
+    "./2015-05-01-preview/swagger/encryptionProtectors.json",
+    "./2017-03-01-preview/swagger/cancelOperations.json"
```
Comment (jaredmoo) [inline arm-sql/compositeSql.json]: Let's not add this until the NET SDK test is ready.

**Comment by fearthecowboy** [comment]
@jiawang003 , @jaredmoo , @dsgouda 

Is this PR nearly ready to merge? I'd like to get the last of the PRs against `master` completed and merged so we're not in this state.

If this is going to take much longer, you may need to create a new PR against  the `current` branch.

**Comment by dsgouda** [comment]
Requested changes have not been addressed yet, cannot merge. @jiawang003 please open this against the current branch as you anyway have to make some changes.

**Comment by fearthecowboy** [comment]
Closing -- please open a new PR against the `current` branch

**Pull request closed without merging**